content/about/board/2022-10-24.md

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
```suggestion
---
layout: page
show_meta: false
title: '2022-10-24'
---

```

Looks like you need this meta info at the beginning of the file to make it render correctly on the website.